streams/examples/main/java/org/apache/kafka/streams/examples/pageview/PageViewTypedJob.java

```diff
@@ -73,6 +77,10 @@ public static void main(String[] args) throws Exception {
         props.put(StreamsConfig.VALUE_SERIALIZER_CLASS_CONFIG, JsonPOJOSerializer.class);
         props.put(StreamsConfig.KEY_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class);
         props.put(StreamsConfig.VALUE_DESERIALIZER_CLASS_CONFIG, JsonPOJODeserializer.class);
+        props.put(StreamsConfig.TIMESTAMP_EXTRACTOR_CLASS_CONFIG, JsonTimestampExtractor.class);
+
+        // can specify underlying client configs if necessary
+        props.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
```

Comment:
For the various instances where we reset to "earliest" (this example but also others) we should also explain why we are actually doing this.  The current comment only tells the reader that, yes, you can also customize the consumer configuration (via put/ConsumerConfig) but we don't explain why we need this specific setting.